src/transformers/models/whisper/feature_extraction_whisper.py

```diff
@@ -307,7 +307,6 @@ def __call__(
             max_length=max_length if max_length else self.n_samples,
             truncation=truncation,
             pad_to_multiple_of=pad_to_multiple_of,
-            **kwargs,
```

Comment:
Was this wrong before? If we don't use the kwargs we can remove them from the call in L288:
https://github.com/huggingface/transformers/blob/d24e84d9edde889bd56ccd46617208cb118d66fa/src/transformers/models/whisper/feature_extraction_whisper.py#L228

Reply:
Yes, the `add_special_tokens` was passed down until here. Checked that `wav2vec2` does not use `kwargs` for padding, plus the call is similar and `kwargs` are not popped out, so appears as the good solution.

Reply:
We might have to keep them for pipeline compatibility, will check that thanks 👍🏻

Reply:
Okay brill - sounds good in that case (pop from here, **not** in L228).